include/window.h

```diff
@@ -1,6 +1,8 @@
 #ifndef GUARD_WINDOW_H
 #define GUARD_WINDOW_H
 
+#define PALETTE_NUM_TO_FILL_VALUE(num) ((num) | ((num) << 4))
```

Comment:
I just realized this is not named properly.  This argument `num` is a pixel value, not a palette, as the define name suggests.  Perhaps `PIXEL_FILL` would be better?

Reply:
Is it? The input numbers seem to reference colors in the current palette. For example, 0 is always used as transparent, and in the menu code 1 always means the background white color.

Reply:
Correct. They are color indexes in the current palette.  A pixel is defined as one of these color indexes.

Reply:
Okay. So rename the whole thing `PIXEL_FILL`?

Reply:
That's what I would do